boards/risc-v/bl808/ox64/src/bl808_appinit.c

```diff
@@ -164,4 +168,14 @@ void board_late_initialize(void)
   mount(NULL, "/proc", "procfs", 0, NULL);
 
 #endif
-}
+
+#ifdef CONFIG_USERLED
+  /* Register the LED driver */
+
+  int ret = userled_lower_initialize("/dev/userleds");
+  if (ret < 0)
+    {
+      syslog(LOG_ERR, "ERROR: userled_lower_initialize() failed: %d\n", ret);
+    }
+#endif
+}
```

Comment:
After the last line of the line: Could you insert a blank line? Just to make nxstyle happy. Thanks :-)

```text
$ ../nxstyle boards/Kconfig
../nxstyle boards/risc-v/bl808/ox64/configs/nsh/defconfig
../nxstyle boards/risc-v/bl808/ox64/include/board.h
../nxstyle boards/risc-v/bl808/ox64/src/Makefile
../nxstyle boards/risc-v/bl808/ox64/src/bl808_appinit.c
../nxstyle boards/risc-v/bl808/ox64/src/bl808_autoleds.c
../nxstyle boards/risc-v/bl808/ox64/src/bl808_userleds.c

boards/risc-v/bl808/ox64/src/bl808_appinit.c:181:0: error: Garbage follows right bracket
```

Reply:
Done!